File: app/src/main/java/com/undancer/app/IndexController.java
package com.undancer.app;

import com.google.common.collect.Maps;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

@RestController
public class IndexController {

    @GetMapping(path = "/")
    public Object get() {
        Map<String, Object> returnObject = Maps.newHashMap();
        returnObject.put("username", "undancer");
        return returnObject;
    }
}

File: app/src/main/kotlin/com/undancer/app/IndexController.kt
package com.undancer.app

import com.google.common.collect.Maps
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class IndexController {

    @GetMapping(path = ["/"])
    fun get(): Any {
        val returnObject = Maps.newHashMap<String, Any>()
        returnObject["username"] = "undancer"
        return returnObject
    }
}
